ladybug/types/angle.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+"""Generic data type."""
+from __future__ import division
+
+from ._base import DataTypeBase
+
+import math
+PI = math.pi
+
+
+class Angle(DataTypeBase):
+    """Angle"""
+    name = 'Angle'
+    units = ['degrees', 'radians']
```

Comment:
The use of lists is critical because the first unit in the list is the base unit that all of the conversion formulas are written in terms of.  This fact gets used in all of the built-in range checks.  If you really want a tuple, I can add another attribute for base_unit which says which of the units is the one in which all of the unit converters are written in.  However, I thought that having a single list was a more elegant solution.

Reply:
All of these have been converted to tuples.